src/components/toggle/toggle.ts

```diff
@@ -0,0 +1,332 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChildren,
+  Directive,
+  EventEmitter,
+  HostBinding,
+  HostListener,
+  Input,
+  OnInit,
+  Optional,
+  Output,
+  QueryList,
+  ViewEncapsulation,
+  forwardRef
+} from '@angular/core';
+import {MdToggleDispatcher} from './toggle_dispatcher';
+
+
+export {MdToggleDispatcher} from './toggle_dispatcher';
+
+
+
+var _uniqueIdCounter = 0;
+
+/** A simple change event emitted by either MdToggle or MdToggleGroup. */
+export class MdToggleChange {
+  source: MdToggle;
+  value: any;
+}
+
+/** Exclusive selection toggle group. */
+@Directive({
+  selector: 'md-toggle-group:not([multiple])',
+  host: {
+    'role': 'radiogroup',
+  },
+})
+export class MdToggleGroup implements AfterContentInit {
+  /** The value for the toggle group. Should match currently selected toggle. */
+  private _value: any = null;
+
+  /** The HTML name attribute applied to toggles in this group. */
+  private _name: string = null;
+
+  /** Disables all toggles in the group. */
+  private _disabled: boolean = null;
+
+  /** The currently selected toggle, should match the value. */
+  private _selected: MdToggle = null;
+
+  /** Event emitted when the group's value changes. */
+  @Output()
+  change: EventEmitter<MdToggleChange> = new EventEmitter<MdToggleChange>();
```

Comment:
The other components still need to be updated to do this, but the `EventEmitter` should be privite while exposing an `Observable` on the public API:
```ts
private _change: EventEmitter<MdToggleChange> = new EventEmitter<MdToggleChange>();
@Output() get change(): Observable<MdToggleChange> { 
  return this._change.asObservable();
}
```